packages/client/src/app/components/challenge/progress/data.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
I think the following structure will be received from the backend:

{
  "data": [
    {
      "id": 1,
      "name": "Challenge 1!",
      "challenge_users": [
        {
          "id": 1,
          "completed": false,
          "user": {
            "id": 1,
            "name": "Marjo"
          }
        },
        {
          "id": 1,
          "completed": false,
          "user": {
            "id": 1,
            "name": "Ahmed"
          }
        }
      ]
    }
  ]
}

Reply:
Actually, should the changes in [this PR](https://github.com/NoNamer777/WA-2022-group-2/pull/20) be approved and merged, the data structure returned from the server should be directly usable instead of nested inside an object. Those changes are of course still up for debate, But imo it should result in easier usage of the data. So the data structure implemented here by @majsa1 should be pretty accurate already

Reply:
What shall we do, pick it up in a new branch?